Cargo.toml

```diff
@@ -21,7 +21,7 @@ default = ["tls"]
 native-tls = { version = "0.2.2", optional = true }
 regex = "1.0"
 bufstream = "0.1"
-imap-proto = "0.14.0"
+imap-proto = "0.14"
```

Comment:
I don't think this should be necessary? Semantic versioning will already take care to use the latest version. If anything this should list the patch version that added any feature you now depend on.

Reply:
Changed :heavy_check_mark: